lib/Mojolicious/Command/eval.pm

```diff
@@ -49,6 +56,7 @@ Mojolicious::Command::eval - Eval command
 =head1 DESCRIPTION
 
 L<Mojolicious::Command::eval> runs code against applications.
+If the result is a L<Mojo::Promise>, it will wait until the promise is completed and the result is returned.
```

Comment:
Before the PR was opened @kraih commented on this line on a commit https://github.com/mojolicious/mojo/commit/4eaf550044e988635d41f4ce0349a2c18acf6b9d#r31802449

Reply:
Do you mean just my added sentence or the existing one as well?

Reply:
Oh! Do you just mean the literal line formatting? Yeah, I'll fix that, sorry.